static/tabs.html

```diff
@@ -8,10 +8,10 @@
         <!-- implicitly includes leaflet, probably -->
         <script src='https://api.tiles.mapbox.com/mapbox.js/v2.0.0/mapbox.js'></script>
         <link href='https://api.tiles.mapbox.com/mapbox.js/v2.0.0/mapbox.css' rel='stylesheet'>
-        <script src="http://code.jquery.com/jquery-1.11.2.min.js"></script>
-        <script src="http://code.jquery.com/ui/1.11.2/jquery-ui.min.js"></script>
+        <script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
```

Comment:
you can also just remove the protocol and it will use the one of the incoming request IIRC. 

`//code.jquery.com/jquery-1.11.2.min.js` will do it